### ISSUE TITLE
Error while installing

### ISSUE THREAD

**Issue by cies** [slamdata/purescript-css#42]
When installing without `--production` I get this error (with that flag, no problems):

```
$ bower install
$ npm run -s build

* Building project in /home/cies/repos/private/purescript-css
[1/1 UnknownImport] bower_components/purescript-psci-support/src/PSCI/Support.purs:10:34

  10  import Control.Monad.Eff.Unsafe (unsafeCoerceEff)
                                       ^^^^^^^^^^^^^^^

  Cannot import value unsafeCoerceEff from module Control.Monad.Eff.Unsafe
  It either does not exist or the module does not export it.
```

I expect it to be an incompatibility between `purescript-psci-support-2.0.0` and the `purescript-eff` lib version that gets indirectly imported by the non-dev deps of `purescript-css`.

This is not caught by Travis' builds because it puts the `--production` flag.

I think Purescript would benefit a system like Haskell's `stack`, and this (type of) error(s) would not have bitten me.

**Comment by cies** [comment]
Lowering the version of `purescript-psci-support` to 1.0.0 fixes this.

**Comment by garyb** [comment]
> I think Purescript would benefit a system like Haskell's stack, and this (type of) error(s) would not have bitten me. 

Soon https://github.com/purescript/purescript/pull/2337 :)

This project hasn't been updated for the round of libraries released with PureScript 0.10 yet, which `purescript-psci-support@2.0.0` is part of. It also doesn't depend on that, so I assume this arose by pulling `purescript-css` in as a dependency elsewhere?

**Comment by garyb** [comment]
This should be fixed now, a new release has been made for PureScript 0.10. The old version is still available for 0.9.x too, so it's covered either way. :smile:
